type-adapters/.gitignore

```diff
@@ -0,0 +1 @@
+/build
```

Comment:
I'm not sure if this is necessary... the gitignore file in the root directory should cover this.